server/pages/index.js

```diff
@@ -27,6 +27,7 @@ var staticFiles = [
 ];
 
 var sections = sectionsModule.get();
+var commitChecksum = getCurrentCommitShortChecksum();
```

Comment:
As @ehg suggested, if we're using this in prod, I'd suggest using exec async or spawn.
```javascript
const exec = require( 'child_process' ).exec;
let commitChecksum;
exec( 'git rev-parse --short HEAD', function( error, stdout, stderr ) {
            if ( error ) {
                return;
            }
            commitChecksum = stdout.toString().replace( /\s/gm, '' );
} );
```